src/modules/src/pid.c

```diff
@@ -90,18 +92,22 @@ float pidUpdate(PidObject* pid, const float measured, const bool updateError)
 
     pid->outI = pid->ki * pid->integ;
     output += pid->outI;
-
-    //filter complete output instead of only D component to compensate for increased noise from increased barometer influence
-    if (pid->enableDFilter)
-    {
-      output = lpf2pApply(&pid->dFilter, output);
-    }
-    else {
-      output = output;
-    }
-    if (isnan(output)) {
-      output = 0;
-    }
+    
+    #ifdef IMPROVED_BARO_Z_HOLD
```

Comment:
Filtering the entire PID output can be useful also in other situation... 
Maybe better to use a different define name?

e.g. PID_FILTER_ALL

and then
#ifdef IMPROVED_BARO_Z_HOLD
#define PID_FILTER_ALL
#endif

Reply:
Yeah true, I agree

Reply:
has been done